ci/deploy.yml

```diff
@@ -116,47 +116,83 @@ connect-explorer deploy dev:
 beta blockchain-link deploy npm:
   <<: *npm_deploy_rules
   script:
-    - nix-shell --run "yarn && yarn workspace @trezor/blockchain-link build:lib && cd ./packages/blockchain-link && npm publish --tag beta"
+    - nix-shell --run "yarn && yarn build:libs && cd ./packages/blockchain-link && npm publish --dry-run && npm publish --tag beta"
```

Comment:
Isn't the `npm publish --dry-run` redundant here? I believe the `npm publish` command has the same output.

But maybe we could add the dry run as separate job just to check? Not sure if it would be useful.

Reply:
removed